policyengine_us/variables/gov/states/ar/tax/income/credits/personal/amount/ar_dependent_credit.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class ar_dependent_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arkansas dependent personal credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2021_AR1000F_FullYearResidentIndividualIncomeTaxReturn.pdf"
+        "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_FullYearResidentIndividualIncomeTaxReturn.pdf#page=1"
+        "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=12"
+    )
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
```

Comment:
if a dependent is disabled, the filer gets an extra $500 per disabled person

- create a parameter with the 500 value 

- check the number of people who are` is_disabled & is_tax_unit_dependent `

- number has to be summed up to the tax_unit using tax_unit.sum

Reply:
need a test with `people:`

Reply:
test failed with 0 output